README.md

```diff
@@ -81,7 +84,7 @@ cd Unvanquished/
 git submodule update --init --recursive
 ```
 
-If cmake complains about the `daemon/` folder being empty then you have skipped this step.
+If cmake complains about missing files in `daemon/` folder or similar issue then you have skipped this step.
```

Comment:
Why not be exact: missing files in `daemon/` or `libs/`

Reply:
If submodules were not fetched at all, the error is:

```
CMake Error at CMakeLists.txt:67 (add_subdirectory):
  The source directory

    /tmp/Unvanquished/daemon

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:68 (include):
  include could not find load file:

    DaemonGame


CMake Error at CMakeLists.txt:115 (try_flag):
  Unknown CMake command "try_flag".


-- Configuring incomplete, errors occurred!
```

Quoing `daemon` looks to be enough.